src/lib/getClaimIds.js

```diff
@@ -1,5 +1,20 @@
+//Claims may have qualifier series ordinal, this should be set when the children birthdate is unknown, but they should still be sorted by age.
+function getSeriesOrdinal(claim) {
+  if (
+    claim.qualifiers &&
+    claim.qualifiers.P1545 &&
```

Comment:
if you can put it in consts

Reply:
then I would need a try and catch, is this now the preferred way?

Reply:
I meant `const SERIES_ORDINAL_ID = "P1545"`